WindowsServerDocs/security/guarded-fabric-shielded-vm/guarded-fabric-tpm-trusted-attestation-capturing-hardware.md

```diff
@@ -135,7 +135,7 @@ For more information about the available CI policy rule levels, see [Deploy code
     ```powershell
     Add-HgsAttestationCIPolicy -Path <Path> -Name '<PolicyName>'
     ```
-
+    [!NOTE] If you're using a signed code integrity policy, register an unsigned copy of the same policy with HGS. The signature on code integrity policies is used to control updates to the policy, but is not measured into the host TPM and therefore cannot be attested to by HGS.
```

Comment:
Please note that the following layout is the MarkDown Note blob format used for the colored Note blobs in the docs.microsoft.com documentation pages:
```suggestion
    > [!NOTE]
    > If you're using a signed code integrity policy, register an unsigned copy of the same policy with HGS. The signature on code integrity policies is used to control updates to the policy, but is not measured into the host TPM and therefore cannot be attested to by HGS.
```

Reply:
… or have I missed something important regarding MD formatting in this context?